.github/ISSUE_TEMPLATE/membership.md

```diff
@@ -23,9 +23,6 @@ e.g. (at)kubernetes
 - (at)sponsor-2
 
 ### List of contributions to the Kubernetes project
-- <PR reviewed / authored>
-- <PR reviewed / authored>
-- <Issue responded to>
-- <Issue responded to>
-- <SIG project I am involved with>
-- <SIG project I am involved with>
+- PRs reviewed / authored
+- Issues responded to
```

Comment:
This doesn't sound as good as before but my markdown-fu is failing me. I'm not sure how to make it sound the same way as but not use crazy markdown. :upside_down_face:

Reply:
From an end user perspective this looks good :)